std/experimental/ndslice/slice.d

```diff
@@ -638,6 +638,16 @@ auto makeSlice(T,
 }
 
 /// ditto
+auto makeSlice(
+    Flag!`replaceArrayWithPointer` replaceArrayWithPointer = Yes.replaceArrayWithPointer,
```

Comment:
@wilzbach please add `replaceArrayWithPointer` for each function in the module and fix its name in the docs for `sliced` (it called `mod` instead of `replaceArrayWithPointer`)